### PR TITLE
Move to beta5

### PR DESCRIPTION
**Pull request by upendras**

### CODE REVIEW

File: test/FunctionalTestUtils/InProcessServer.cs [hunk 1]
```diff
@@ -4,14 +4,24 @@
     using Microsoft.ApplicationInsights.Channel;
     using Microsoft.AspNet.Hosting;
     using Microsoft.AspNet.Server.WebListener;
-    using Microsoft.Framework.ConfigurationModel;
+    using Microsoft.Framework.Configuration;
     using Microsoft.Framework.DependencyInjection;
     using Microsoft.Framework.Logging;
+    using Microsoft.AspNet.Hosting.Internal;
```
Comment (olegsych) [inline test/FunctionalTestUtils/InProcessServer.cs]: Should we "remove and sort" usings here?

File: test/FunctionalTestUtils/project.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+        "Microsoft.AspNet.Hosting": "1.0.0-beta5-*",
+        "Microsoft.AspNet.Http.Abstractions": "1.0.0-beta5-*",
+        "Microsoft.AspNet.Mvc": "6.0.0-beta5-*",
+        "Microsoft.AspNet.Server.WebListener": "1.0.0-beta5-*",
+        "Microsoft.Framework.Runtime": "1.0.0-beta5-*",
         "Microsoft.Net.Http.Client": "1.0.0-*",
-        "xunit": "2.1.0-*",
-        "xunit.runner.dnx": "2.1.0-*",
-        "xunit.runner.visualstudio": "2.1.0-*"
+        "xunit.runner.aspnet": "2.0.0-aspnet-*"
```
Comment (olegsych) [inline test/FunctionalTestUtils/project.json]: Is this right? We are going to an older version of the dnx test runner?

Comment (upendras) [inline test/FunctionalTestUtils/project.json]: Test explorer doesn't seem to work with dnx runner still :(

File: test/Microsoft.ApplicationInsights.AspNet.Tests/Extensions/HttpRequestExtensionsTest.cs [hunk 3]
```diff
@@ -2,7 +2,7 @@
 {
     using Microsoft.ApplicationInsights.AspNet.Extensions;
     using Microsoft.AspNet.Http;
-    using Microsoft.AspNet.Http.Core;
+    using Microsoft.AspNet.Http.Internal;
```
Comment (olegsych) [inline test/Microsoft.ApplicationInsights.AspNet.Tests/Extensions/HttpRequestExtensionsTest.cs]: What are we using from the `Http.Internal` namespace? Is it safe for us to continue doing so?

File: test/Microsoft.ApplicationInsights.AspNet.Tests/project.json [hunk 4]
```diff
@@ -1,26 +1,23 @@
...
+    "Microsoft.AspNet.Mvc.Core": "6.0.0-beta5-*",
+    "Microsoft.Framework.Logging": "1.0.0-beta5-*",
+    "Microsoft.Framework.Configuration.Json": "1.0.0-beta5-*",
+    "xunit.runner.aspnet": "2.0.0-aspnet-*"
   },
   "commands": {
-    "test": "xunit.runner.dnx"
+    "web": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.WebListener --server.urls http://localhost:5000",
+    "gen": "Microsoft.Framework.CodeGeneration",
+    "ef": "EntityFramework.Commands",
```
Comment (olegsych) [inline test/Microsoft.ApplicationInsights.AspNet.Tests/project.json]: do we need the web, gen and ef commands? Are we referencing the `Microsoft.Framework.CodeGeneration` and `EntityFramework.Commands` packages where the `gen` and `ef` commands are defined?

File: test/Microsoft.ApplicationInsights.AspNet.Tests/Extensions/HttpRequestExtensionsTest.cs (hunk 3 above)
Comment (upendras) [inline test/Microsoft.ApplicationInsights.AspNet.Tests/Extensions/HttpRequestExtensionsTest.cs]: DefaultHttpContext used in the test cases is from this namespace. this should be fine IMO

File: test/Microsoft.ApplicationInsights.AspNet.Tests/project.json (hunk 4 above)
Comment (upendras) [inline test/Microsoft.ApplicationInsights.AspNet.Tests/project.json]: note required, will remove.

File: test/FunctionalTestUtils/project.json (hunk 2 above)
Comment (olegsych) [inline test/FunctionalTestUtils/project.json]: I got it to work on one of my [hobby projects](https://github.com/olegsych/dice/blob/develop/test/Dice.Abstractions.Tests/project.json). I remember having to restore packages and close VS a couple of times.

Comment (upendras) [inline test/FunctionalTestUtils/project.json]: sure. let me give it a shot one more time.

Comment (upendras) [inline test/FunctionalTestUtils/project.json]: turns out https://github.com/xunit/dnx.xunit had its last release 2.1 Beta 2 on 5/3 and that supported only ASP.Net 5 beta4. There is a recent commit moving the current branch to beta5, but that isn't released yet. Wondering if we should wait?

Comment (upendras) [inline test/FunctionalTestUtils/project.json]: created #88 for the same.

**Comment by olegsych** [comment]
:shipit:

**Pull request merged**